Make-arch

```diff
@@ -0,0 +1,1732 @@
+# Makefile architecture configuration for Tachyon ray tracing library.
+# Copyright 1994-2011 John E. Stone
```

Comment:
Why are we adding this Makefile? Is this something they include in their source, which we should be patching?